astropy/time/core.py

```diff
@@ -1043,6 +1205,7 @@ def min(self, axis=None, out=None, keepdims=False):
         ``np.min``; since `Time` instances are immutable, it is not possible
         to have an actual ``out`` to store the result in.
         """
+
```

Comment:
Bit of a drive-by comment, but why the extra blank lines here and below? Blank lines after docstrings are only suggested for modules and classes by PEP 257 - https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
No strong reason.  I've been influenced a bit by the [plenty-of-vertical-whitespace style](https://github.com/astrofrog/fast-histogram/blob/master/fast_histogram/histogram.py) of @astrofrog, which often includes a blank line after the docstring.  Also it somehow feels just a little easier to see the first code line after a multiline docstring with that space.  I don't quite follow the distinction that PEP-257 makes between functions and class methods.

Reply:
On the [Numpy doc guide](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) the blank line is always before the quotes, not after. So that's what I use in my projects, but not sure if there is a real recommendation about this.

Reply:
I include a blank line after docstrings (for both classes and functions; IMHO it makes things more readable).  Several people put a blank line before the end quotes, but I've never understood the reason for that or knew where it came from (it's not in PEP-257).

Reply:
I always felt the closing three quotes on a line make for a clear separation - and there is also a benefit to having more code on a single screen-full.

In any case, it probably ends up a matter of taste, so I'd say that we follow the general rule to not stick with what a previous person put there unless it clearly violates pep8/pep257 (and definitely not change away from pep8/pep257).

(I write this trying to restrain myself every time I see lines that are longer than 80 characters and which thus wrap in my editor).

Reply:
I agree fully with not changing the style of existing code and minimizing changes.  Like many public figures these days, I don't actually remember doing this.  But I do have the option of reverting this past mistake and will do so.  :smile: